components/mediation-ui/org.wso2.carbon.sequences.ui/src/main/java/org/wso2/carbon/sequences/ui/client/SequenceAdminClient.java

```diff
@@ -176,7 +176,7 @@ public void addSequence(SequenceMediator sequence) throws SequenceEditorExceptio
         } catch (Exception e) {
             if (e.getMessage().toString().toLowerCase().contains("this name already exists".toLowerCase())) {
                 //Error Related to name already exits
-                handleException(e.getMessage(), e);
+                throw new SequenceEditorException(e.getMessage(), e);
```

Comment:
Removing the handleException and manually throwing the exception, Since it will result of printing redundant exception trace which was already being printed.